skype/skype-ps/skype/New-CsTeamsMeetingBrandingPolicy.md

```diff
@@ -0,0 +1,196 @@
+---
+external help file: Microsoft.Teams.Policy.Administration.Cmdlets.Core.dll-Help.xml
+Module Name: MicrosoftTeams
+online version: https://learn.microsoft.com/powershell/module/skype/new-csteamsmeetingbrandingpolicy
+schema: 2.0.0
+title: New-CsTeamsMeetingBrandingPolicy
+author: szymonkatraMSFT
+ms.author: szymonkatra
+ms.reviewer:
+manager: stanlythomas
+---
+
+# New-CsTeamsMeetingBrandingPolicy
+
+## SYNOPSIS
+The CsTeamsMeetingBrandingPolicy cmdlets enable administrators to control the appearance in meetings by defining custom background, logo and colors.
+
+## SYNTAX
+
+```
+New-CsTeamsMeetingBrandingPolicy
+ [-MeetingBackgroundImages <System.Management.Automation.PSListModifier`1[Microsoft.Teams.Policy.Administration.Cmdlets.Core.MeetingBackgroundImage]>]
+ [-MeetingBrandingThemes <System.Management.Automation.PSListModifier`1[Microsoft.Teams.Policy.Administration.Cmdlets.Core.MeetingBrandingTheme]>]
```

Comment:
```suggestion
 [-MeetingBrandingThemes <PSListModifier>]
```

Reply:
Please specify the object type this way also for others

Reply:
Done